explorer/interpreter/interpreter.cpp

```diff
@@ -453,6 +453,20 @@ auto Interpreter::StepLvalue() -> ErrorOr<Success> {
         return todo_.FinishAction(arena_->New<LValue>(field));
       }
     }
+    case ExpressionKind::BaseAccessExpression: {
+      const auto& access = cast<BaseAccessExpression>(exp);
+      if (act.pos() == 0) {
+        //    { {e.base :: C, E, F} :: S, H}
```

Comment:
We haven't been requiring these as we go along, instead preferring plain english explanations. Do what you want here, if I feel strongly then maybe at some point I'll remove all of them because I assume they mostly get added because people see them in a lot of spots. :)

Reply:
Switched to "plain english"